app/Models/Aircraft/WakeCategory.php

```diff
@@ -32,15 +32,24 @@ public function toArray()
             'id' => $this->id,
             'code' => $this->code,
             'description' => $this->description,
-            'subsequent_departure_intervals' => $this->departureIntervals->sortBy('relative_weighting')->map(
-                function (WakeCategory $subsequent) {
-                    return [
-                        'id' => $subsequent->id,
-                        'interval' => $subsequent->pivot->interval,
-                        'intermediate' => $subsequent->pivot->intermediate,
-                    ];
-                }
-            )->values()->toArray(),
+            'subsequent_departure_intervals' => $this->departureIntervals
+                ->sortBy('relative_weighting')
+                ->map(
+                    function (WakeCategory $subsequent) {
+                        return [
+                            'id' => $subsequent->id,
+                            'interval' => $subsequent->pivot->interval,
```

Comment:
Is there a way to eager load this pivot? As I see it, you could have an N + 1 issue here each time you try to access the pivot.

Reply:
Eager loaded where it's used - in WakeService line 11 :)